assets/js/stats/unique.js

```diff
@@ -0,0 +1,33 @@
+export function updateDailyUnique( key ) {
+	const date = new Date();
+	const expiresAtTimestamp = date.getTime() + 24 * 60 * 60 * 1000;
+	window.localStorage[key] = expiresAtTimestamp;
+}
+
+export function getDailyUnique( name ) {
+	if ( window.localStorage[name] ) {
+		const date = new Date();
+		const now  = date.getTime();
+		const expiration = parseInt( window.localStorage[ name ], 10 );
+		return expiration >= now;
```

Comment:
We should check if it parsed to a number, otherwise this seems to be always false.